packages/api/src/models/Transaction.js

```diff
@@ -25,6 +29,17 @@ module.exports = class Transaction {
 
   // eslint-disable-next-line camelcase
   static fromRow ({ tx_hash, index, block_hash, block_height, type, data, timestamp, gas_used, status, error }) {
-    return new Transaction(tx_hash, index, block_hash, block_height, type, data, timestamp, parseInt(gas_used), status, error)
+    let decodedError = null
+
+    try {
+      if (typeof error === 'string') {
```

Comment:
Handle all cases including DAPI errors and Protocol errors. You can find an example in dapi-grpc